### PR TITLE
Feature/tvist1 dashboard grid update

### PR DESCRIPTION
**Pull request by jekuaitk**
* Updated grid on dashboard with active hearing
* Avoids counting unimplemented counts on grid

### CODE REVIEW

File: src/Form/CaseFilterType.php [hunk 1]
```diff
@@ -213,6 +213,11 @@ function (FormEvent $event) use ($formModifier) {
                         switch ($filterChoice) {
                             case CaseSpecialFilterStatuses::IN_HEARING:
                                 // TODO: When hearing implemented: modify query builder correctly with cases having an active hearing
+                                $qb = $filterQuery->getQueryBuilder();
```
Comment (rimi-itk) [inline src/Form/CaseFilterType.php:216]: Can the “TODO” line above this be removed now?

File: src/Service/DashboardHelper.php [hunk 2]
```diff
@@ -80,11 +80,20 @@ private function getUserColumnCaseInformation(Municipality $municipality, User $
             'count' => $this->caseRepository->findCountOfCasesWithSomeExceededDeadlineBy(['municipality' => $municipality, 'assignedTo' => $user]),
         ];
 
-        $count = array_sum(array_column($row, 'count'));
+        // TODO: When all rows are fully implemented update beneath
+//        $count = array_sum(array_column($row, 'count'));
```
Comment (rimi-itk) [inline src/Service/DashboardHelper.php:84]: Remove this line (and maybe the “TODO” above).

File: src/Service/DashboardHelper.php [hunk 3]
```diff
@@ -150,11 +159,20 @@ private function getBoardColumnsCaseInformation(Municipality $municipality): arr
                 'count' => $this->caseRepository->findCountOfCasesWithSomeExceededDeadlineBy(['board' => $board]),
             ];
 
-            $boardCount = array_sum(array_column($rows, 'count'));
+            // TODO: When all rows are fully implemented update beneath
+//            $boardCount = array_sum(array_column($rows, 'count'));
```
Comment (rimi-itk) [inline src/Service/DashboardHelper.php:163]: Remove this line (and maybe the “TODO” above).

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: src/Form/CaseFilterType.php (hunk 1 above)
Comment (jekuaitk) [reply src/Form/CaseFilterType.php:215]: Removed in c22aeb68fb47a45b67c0be2f7a44b145efa208dc.

File: src/Service/DashboardHelper.php (hunk 2 above)
Comment (jekuaitk) [reply src/Service/DashboardHelper.php]: Reverted back to before in c22aeb68fb47a45b67c0be2f7a44b145efa208dc and hid unimplemented rows in dashboard grid instead.

File: src/Service/DashboardHelper.php (hunk 3 above)
Comment (jekuaitk) [reply src/Service/DashboardHelper.php]: Reverted this change in c22aeb68fb47a45b67c0be2f7a44b145efa208dc and hid unimplemented rows in dashboard grid instead.

**Pull request merged**